make.js

```diff
@@ -317,7 +299,7 @@ target.build = function() {
 // node make.js test
 // node make.js test --task ShellScript --suite L0
 //
-target.test = function() {
+CLI.test = function(argv) {
```

Comment:
This is just a nice-to-have but if you like the idea, consider adding JSDoc types to the argument so that the TypeScript language service in VS Code (which does type inference for JavaScript in VS Code) can have more information to populate Intellisense:

`function(/** @type {{ suite: string; node: string; … }} */ argv)`